content/project/governance.adoc

```diff
@@ -0,0 +1,320 @@
+
+* link:https://raw.github.com/jenkinsci/infra-cla/approved/icla.txt[ICLA]
+* link:https://raw.github.com/jenkinsci/infra-cla/approved/ccla.txt[CCLA]
+
+To balance the additional overhead with the lower barrier to entry, this policy only applies to the core, although we welcome anyone who's willing to submit a CLA for their work on plugins.
+
+Details to submit CLAs are documented in link:https://github.com/jenkinsci/infra-cla/blob/master/README.md[infra-cla README].
+
+=== 3rd party library licenses in the core
+
+The Jenkins project depends heavily on 3rd party libraries. We believe it's better to reuse what we can than to reinvent, so that our precious resources can be spent elsewhere.
+
+Jenkins as a whole needs to be an open-source project, so we've been restricting ourselves to only use 3rd party libraries that are link:http://www.opensource.org/[OSI-approved open-source licenses] (such as Apache Software License, BSD license, MIT license, Eclipse Public License, and Lesser GNU Public License.) In addition, to keep the core reusable for the widest possible use cases, we refuse so-called link:http://en.wikipedia.org/wiki/Copyleft[copyleft licenses] that restrict the overall license to a specific license (such as GNU Public License.) To that end, we designated the following licenses as approved licenses that we can depend on. Additional licenses are subject to the approval by the board:
+
+* link:http://www.apache.org/licenses/[Apache Software License]
+* link:http://opensource.org/licenses/bsd-license.php[BSD license], link:http://opensource.org/licenses/mit-license.php[MIT license]
+* link:http://opensource.org/licenses/W3C.php[W3C software license]
+* link:http://opensource.org/licenses/cddl1.php[CDDL], link:http://opensource.org/licenses/cpl1.0.php[CPL], link:http://opensource.org/licenses/eclipse-1.0.php[EPL], and link:http://opensource.org/licenses/mozilla1.0.php[MPL], and link:http://opensource.org/licenses/lgpl-license.php[LGPL] (these licenses require that derivative works to be under the same license as the original license, so the source code needs to be separated into its own repository, but binaries can be used as dependencies to the core.)
+
+In core, the build process ensure that all the licenses of the dependency libraries are accounted for. In other words, we require that dependency POMs declare their own licenses in the Maven way, or use link:https://github.com/jenkinsci/jenkins/blob/master/licenseCompleter.groovy[the license completion script] to supply one where the POM is missing that information. The completion script is also used in case the library is dual-licensed to pick a specific license. This is done so that the summary list of the 3rd party library licenses do not include copy-left licenses.
+
+The result is packaged into the core and is visible from link:https://ci.jenkins-ci.org/about[within the application itself]. It is also available in the XML format inside the war file.
```

Comment:
I know that most links are out of date in this document (i.e. don't point to jenkins.io, or point to SVN which is dead), but all the redirects work fine for the old website, so there's no big rush to change them.

However, I think https://ci.jenkins-ci.org/ should preferably be changed to https://ci.jenkins.io/ as the current link gives an SSL certificate warning.